### DISCUSSION TITLE
When you are trying to change column type to float (with default precision) - column type will be DOUBLE instead.

### DISCUSSION THREAD

**Discussion by kekemeke** [laravel/framework#52226] [Q&A]
### Laravel Version

11.0.8

### PHP Version

8.2.21

### Database Driver & Version

MySQL 8.0.33-25.1

### Description

First of all - i known, its not a big deal, but...

So, i think if someone use $table->float('column_name') he really want to get FLOAT SQL type, cuz if you want DOUBLE, just use $table->double('column_name') instead.

I mean, why $table->float() has precision=53 as default? instead of 24?

### Steps To Reproduce

1. create $table->integet('name');
2. use $table->float('name')->change().

**Comment by hafezdivandari** [comment]
You may check PR #48861 for the full explaination.
